awx/main/scheduler/task_manager.py

```diff
@@ -485,6 +459,7 @@ def generate_dependencies(self, undeped_tasks):
     def process_pending_tasks(self, pending_tasks):
         running_workflow_templates = {wf.unified_job_template_id for wf in self.get_running_workflow_jobs()}
         tasks_to_update_job_explanation = []
+        controlplane_ig = InstanceGroup.objects.get(name='controlplane')
```

Comment:
We're closing on finishing this up, so let me start with more nit-picky style stuff.

We should use `DEFAULT_CONTROL_PLANE_QUEUE_NAME` instead of the string "controlplane".

Also, I think you should hack this by adding onto `self` in `after_lock_init`. It does a loop over all instance groups, and you can just save this if the name matches this setting. Then you can reference `self.controlplane_ig` or whatever.

Reply:
done